.bazelversion

```diff
@@ -1 +0,0 @@
-4.2.0
```

Comment:
Should this actually be pinned so that CI doesn't spontaneously break?

Reply:
I removed it by accident, while testing.